cockatrice/src/interface/widgets/tabs/api/commander_spellbook/api_response/commander_spellbook_bracket_tag.h

```diff
@@ -0,0 +1,79 @@
+#ifndef COCKATRICE_COMMANDER_SPELLBOOK_BRACKET_TAG_H
+#define COCKATRICE_COMMANDER_SPELLBOOK_BRACKET_TAG_H
+#include <QString>
+
+namespace CommanderSpellbookBracketTag
+{
+enum class BracketTag
+{
+    Ruthless,
+    Spicy,
+    Powerful,
+    Oddball,
+    PreconAppropriate,
+    Casual,
+    Unknown
+};
+
+inline static BracketTag bracketTagFromString(const QString &s)
+{
+    if (s == "R")
+        return BracketTag::Ruthless;
+    if (s == "S")
+        return BracketTag::Spicy;
+    if (s == "P")
+        return BracketTag::Powerful;
+    if (s == "O")
+        return BracketTag::Oddball;
+    if (s == "PA")
+        return BracketTag::PreconAppropriate;
+    if (s == "C")
+        return BracketTag::Casual;
+    return BracketTag::Unknown;
+}
+
+inline static QString bracketTagToString(BracketTag tag)
+{
+    switch (tag) {
+        case BracketTag::Ruthless:
+            return "Ruthless";
+        case BracketTag::Spicy:
+            return "Spicy";
+        case BracketTag::Powerful:
+            return "Powerful";
+        case BracketTag::Oddball:
+            return "Oddball";
+        case BracketTag::PreconAppropriate:
+            return "Precon Appropriate";
+        case BracketTag::Casual:
+            return "Casual";
+        case BracketTag::Unknown:
+            return "Unknown";
+    }
+    return {};
+}
+
+inline static QString bracketTagToOfficialString(BracketTag tag)
+{
+    switch (tag) {
+        case BracketTag::Ruthless:
+            return "[5] cEDH";
```

Comment:
I mentioned in discord how this is a bad idea as the bracket system is much more flexible than this and isn't determined by a static set of rules but rather how consistent a deck is and how well it performs, which is too complex to decide algorithmically

Reply:
I also have the same concern that these bracket tag names could be a bad idea to hard code into the system. What happens if the bracket system suddenly shifts? We'd have outdated terminology with no easy way to push a fix to users out there.

I'm not saying I like the idea of keeping a global data store somewhere on the Cockatrice website, but maybe it makes sense to do that similar to the public servers list? https://github.com/Cockatrice/cockatrice.github.io/blob/master/public-servers.json